src/Microsoft.ML.OnnxConverter/OnnxExportExtensions.cs

```diff
@@ -26,13 +37,23 @@ internal static ModelProto ConvertToOnnxProtobuf(this ModelOperationsCatalog cat
         {
             var env = catalog.GetEnvironment();
             var ctx = new OnnxContextImpl(env, "model", "ML.NET", "0", 0, "machinelearning.dotnet", OnnxVersion.Stable);
-            var outputData = transform.Transform(inputData);
-            LinkedList<ITransformCanSaveOnnx> transforms = null;
-            using (var ch = env.Start("ONNX conversion"))
-            {
-                SaveOnnxCommand.GetPipe(ctx, ch, outputData, out IDataView root, out IDataView sink, out transforms);
-                return SaveOnnxCommand.ConvertTransformListToOnnxModel(ctx, ch, root, sink, transforms, null, null);
-            }
+            return ConvertToOnnxProtobufCore(env, ctx, transform, inputData);
+        }
+
+        /// <summary>
+        /// Convert the specified <see cref="ITransformer"/> to ONNX format. Note that ONNX uses Google's Protobuf so the returned value is a Protobuf object.
+        /// </summary>
+        /// <param name="catalog">The class that <see cref="ConvertToOnnxProtobuf(ModelOperationsCatalog, ITransformer, IDataView, int)"/> attached to.</param>
+        /// <param name="transform">The <see cref="ITransformer"/> that will be converted into ONNX format.</param>
+        /// <param name="inputData">The input of the specified transform.</param>
+        /// <param name="opSetVersion">The OpSet version to use for exporting the model. This value must be greater than or equal to 9 and less than or equal to 12</param>
```

Comment:
Since this is the internal doc for the internal method, this won't actually be exposed to our users. It's better to have this description of opSetVersion inside the docs of the piblic ConvertToOnnx method down below.

Reply:
added